00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+# Agile Sprint Process for Enarx
+- Authors: [Mark Bestavros](mbestavr@redhat.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-05-06
+- Status Note: Concept brainstormed, awaiting implementation
+- Supersedes: N/A
+- Start Date: 2020-05-01
+- Tags: process, sprint, concept
+
+## Summary
+
+Enarx has been using its daily standup and board organization for a while. It's
+worked for us as a small team, but as we grow, we anticipate it becoming
+unsustainable. It also doesn't do a good job capturing the current thrust of our
+work. This RFC proposes a replacement process, designed around the agile sprint
+methodology.
+
+## Motivation
+
+The Enarx project needs to organize its work better, and in a more sustainable
+way than before. We need to do a better job planning for medium-term work and
+goals, and in a way that scales more effectively than our prior process.
+
+### Tracking Work
+
+As a Github-native project, Enarx categorizes its work with the two major
+primitives available: **issues** and **pull requests** (sometimes abbreviated
+PRs).
+
+- **Issues** on Github take the form of a simple text note in Markdown. Each one
+  has a title and can be assigned metadata such as labels and assignees.
+  
+  They represent work that needs to be planned, implemented, or otherwise
+  completed. An issue may encapsulate any task related to the project, and
+  scope can vary wildly from one issue to the next; where one may be a simple
+  formatting fix, others might be a radical design change. Presently (and for
+  the foreseeable future), the Enarx project uses them exclusively to organize
+  tasks yet to be done, discussion topics, and more (even when not strictly
+  needed).
```

Comment:
To be accepted, an issue **must** be well-scoped, and have a definition of "done".  Those writing issues are encouraged to specify or suggests tests to help the assignees to complete the task as expected.

Each issue should also be small enough to be accomplished within one Sprint: if not, it should be decomposed.  In the rare situations where this is not possible, sign-off by the Product Owner is required to spread it over multiple Sprints.

Reply:
I've added a section under "Planning" called "Acceptance Criteria" for this in my latest push.